en/protocol/mission.md

```diff
@@ -1,17 +1,41 @@
 # Mission Protocol
 
-The mission protocol is a sub-protocol supporting guaranteed delivery of messages. It allows to transfer a mission over a lossy link.
+The mission sub-protocol allows a GCS or developer API to manage missions on a drone/component: upload missions, download missions, clear missions, set/get the current mission item, and get change notification.
 
-## MAVLink 2 Extensions
+The protocol follows the client/server pattern, where operations (and most commands) are initiated by the GCS/developer API (client) and acknowledged by the autopilot (server).
 
-MAVLink 2 supports several different mission types in the [MISSION_COUNT](../messages/common.md#MISSION_COUNT) and [MISSION_REQUEST](../messages/common.md#MISSION_REQUEST) messages using the `mission_type` field. This instructs the autopilot to read and write to different lists containing the regular mission ([MAV_MISSION_TYPE_MISSION](../messages/common.md#MAV_MISSION_TYPE_MISSION)), geofence ([MAV_MISSION_TYPE_FENCE](../messages/common.md#MAV_MISSION_TYPE_FENCE)) and safe points used as alternate landing sites ([MAV_MISSION_TYPE_RALLY](../messages/common.md#MAV_MISSION_TYPE_RALLY)).
+The protocol supports re-request of messages that have not arrived, allowing missions to be reliably transferred over a lossy link. <!-- not quite guaranteed :-) -->
 
-On MAVLink 1 links the mission type defaults to `MISSION` and the ground control station should not attempt to send geofence or alternate landing spot coordinates.
 
+## Mission Types {#mission_types}
 
-## Upload a Mission to the Vehicle
+MAVLink 1 supports only "regular" missions.
```

Comment:
Tried to make this section more clear. I "assume" that if you use the MAVLink 2 library but specify MAVLink 1 the mission type you enter will be ignored. But in either case, the protocol expects you to ignore it, because it doesn't "exist" for MAVLink 1.